src/index.js

```diff
@@ -6,13 +6,13 @@ import 'typeface-roboto-condensed';
 import 'typeface-montserrat';
 import './styles/styles.scss';
 import * as serviceWorker from './serviceWorker';
-import Header from '../components/Layout/Header';
-import Footer from '../components/Layout/Footer';
+import Header from './components/Layout/Header';
+import Footer from './components/Layout/Footer';
 import AppRouter from './routes/AppRouter';
 
-ReactDOM.render(<Header />, document.getElementById('Header'));
+ReactDOM.render(<Header />, document.getElementById('header'));
 ReactDOM.render(<AppRouter />, document.getElementById('Main'));
```

Comment:
Should we also change `document.getElementById('Main'));`? from `Main` to `main`?

Reply:
good point!